lib/livebook/intellisense/identifier_matcher.ex

```diff
@@ -131,13 +131,13 @@ defmodule Livebook.Intellisense.IdentifierMatcher do
   The function returns range of columns where the identifier
   is located and a list of matching identifier items.
   """
-  @spec locate_identifier(String.t(), pos_integer(), Intellisense.context()) ::
+  @spec locate_identifier(String.t(), pos_integer(), Intellisense.context(), node()) ::
           %{
             matches: list(identifier_item()),
             range: nil | %{from: pos_integer(), to: pos_integer()}
           }
-  def locate_identifier(line, column, intellisense_context) do
-    case Code.Fragment.surround_context(line, {1, column}) do
+  def locate_identifier(line, column, intellisense_context, node) do
+    case :erpc.call(node, Code.Fragment, :surround_context, [line, {1, column}]) do
```

Comment:
I don't think those calls need to be RPCed, because they work with the code and they don't use any knowledge about the current environment. :)